### PR TITLE
Modification in the implementation of code in the following DGenTemplates files 1) ConditionalSearch.aspx 2) SearchAndUpdate.aspx 3) Detail.aspx.cs and Detail.aspx.vb 4) SearchAndUpdate.aspx.cs and SearchAndUpdate.aspx.vb  Did proper alignment of code in _3TierEngine.cs file.

### PR DESCRIPTION
**Pull request by SandeepNayak077**

### CODE REVIEW

File: root/files/tools/DGenTemplates/SearchAndUpdate.aspx.cs [hunk 1]
```diff
@@ -246,7 +246,7 @@ protected string UOC_btnBatUpd_Click(FxEventArgs fxEventArgs)
         // ControlComment:LoopEnd-PKColumn
 
         // タイムスタンプ列
-        // ・・・
+        parameterValue.AndEqualSearchConditions.Add("_TimeStampColName_", "");
```
Comment (daisukenishino) [inline root/files/tools/DGenTemplates/SearchAndUpdate.aspx.cs]: This modification requires the following branch.

- If the time stamp is not specified  
Output as follows:  
// ・・・
- If the time stamp is specified  
Output as follows:  
parameterValue.AndEqualSearchConditions.Add ("time stamp name", "");

Comment (daisukenishino) [inline root/files/tools/DGenTemplates/SearchAndUpdate.aspx.cs]: Therefore, not completed only modify the template.  
Modification of the tool is also necessary.

File: root/files/tools/DGenTemplates/SearchAndUpdate.aspx.vb [hunk 2]
```diff
@@ -235,7 +235,7 @@ Partial Public Class _TableName_SearchAndUpdate
         ' ControlComment:LoopEnd-PKColumn
 
         ' タイムスタンプ列
-        ' ・・・
+        parameterValue.AndEqualSearchConditions.Add("_TimeStampColName_", "")
```
Comment (daisukenishino) [inline root/files/tools/DGenTemplates/SearchAndUpdate.aspx.vb]: Ditto

**Comment by daisukenishino** [comment]
Please tell us the future of the corresponding plan for this modification.

**Comment by SandeepNayak077** [comment]
Thank you Nishino-san for your valuable comments.

HODC will consider this task in the current sprint and start working on it, because of high priority(2).

HODC will work on 'UI improvement of SPA sample' review comments in Sprint#24.

Please let us know.

File: root/files/tools/DGenTemplates/SearchAndUpdate.aspx.cs (hunk 1 above)
Comment (SandeepNayak077) [inline root/files/tools/DGenTemplates/SearchAndUpdate.aspx.cs]: To achive the above scenario.

HODC thinks that there must need a placeholder like as follows
<add key="CcIsRequired_TimeStamp" value="ControlComment:IsRequired-TimeStamp"/>

Please let us know.

**Comment by daisukenishino2** [comment]
English tutorial will finish at the end of March.
Please refer to the this tutorial to accomplish this task.

**Comment by SandeepNayak077** [comment]
HODC has already did code modification regarding this task.

HODC requested to close this pull request to create new pull request.

**Pull request closed without merging**